utility_analysis_new/combiners.py

```diff
@@ -201,25 +225,9 @@ class CountUtilityAnalysisMetrics:
     noise_kind: pipeline_dp.NoiseKind
 
 
-@dataclass
-class UtilityAnalysisCountAccumulator:
-    count: int
-    per_partition_error: int
-    expected_cross_partition_error: float
-    var_cross_partition_error: float
-
-    def __add__(self, other):
-        return UtilityAnalysisCountAccumulator(
-            self.count + other.count,
-            self.per_partition_error + other.per_partition_error,
-            self.expected_cross_partition_error +
-            other.expected_cross_partition_error,
-            self.var_cross_partition_error + other.var_cross_partition_error)
-
-
-class UtilityAnalysisCountCombiner(pipeline_dp.Combiner):
+class UtilityAnalysisCountCombiner(UtilityAnalysisCombiner):
     """A combiner for utility analysis counts."""
-    AccumulatorType = UtilityAnalysisCountAccumulator
+    AccumulatorType = Tuple[int, int, float, float]
```

Comment:
I think it is important to document what those ints and floats mean somewhere (e.g. maybe here), so something like:

`[count, per_partition_error, expected_cross_partition_error, var_cross_partition_error]`

(also for other aggregations)